clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -277,12 +285,22 @@ export default async function handler(
     `
       : ""
   }${
-    e2eQuery === "true"
+    e2eQuery === "true" // let Cypress set the config and initialize fides.js
       ? ""
       : `
-        // Initialize fides.js with custom config
     var fidesConfig = ${fidesConfigJSON};
-    window.Fides.init(fidesConfig);`
+    ${
+      initializeQuery === "false"
+        ? `// Capture the base config for later initialization
+        window.Fides.config = fidesConfig;`
+        : `// Initialize fides.js with custom config
+        window.Fides.init(fidesConfig);`
+    }
+    ${
+      environment.settings.DEBUG && initializeQuery === "false"
```

Comment:
This DEBUG setting is misleading you - it's meant to be for FidesJS (it's passed along as a global setting to the initialize config) so we shouldn't also use it for Privacy Center. It's poorly named and documented ☺️

For other debug logging I use NODE_ENV === "development". I noticed some inconsistencies with this in my WIP branch adding some more debug logging...

Reply:
oh! much appreciated. I'll swap that out.